src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -1784,24 +1793,17 @@ protected override void ProcessRecord()
                         {
                             if (!service.Status.Equals(ServiceControllerStatus.Stopped))
                             {
-                                //check for the dependent services as set-service dont have force parameter
+                                //check for the dependent services
                                 ServiceController[] dependentServices = service.DependentServices;
 
-                                if ((dependentServices != null) && (dependentServices.Length > 0))
+                                if ((!Force) && (dependentServices != null) && (dependentServices.Length > 0))
                                 {
                                     WriteNonTerminatingError(service, null, "ServiceHasDependentServicesNoForce", ServiceResources.ServiceHasDependentServicesNoForce, ErrorCategory.InvalidOperation);
                                     return;
                                 }
 
-                                ServiceController[] servicedependedon = service.ServicesDependedOn;
-
-                                if ((servicedependedon != null) && (servicedependedon.Length > 0))
-                                {
-                                    WriteNonTerminatingError(service, null, "ServiceIsDependentOnNoForce", ServiceResources.ServiceIsDependentOnNoForce, ErrorCategory.InvalidOperation);
```

Comment:
We should remove the resource string from RESX file  it is no longer used.

Reply:
Hi iSazonov, I am a new guy, can you please tell me I should create a new pull request or I can add new code into this pull request?

Reply:
The new code is committed.